### ISSUE TITLE
Feature request: Combined destination/intermediate time and distance widgets

### ISSUE THREAD

**Issue by martenjj** [osmandapp/OsmAnd#15309]
### Description

Four "Navigation points" widgets are available to be configured to be shown: distance to destination, distance to intermediate, time to destination and time to intermediate.  They are all useful, but if all four are shown while navigating then they take up a lot of space wherever they are placed.

My suggestion is to have 2 more combined widgets available, showing the distance and time either to the next intermediate, or the final destination, depending on what will arrive next.  For example, if starting navigation with one intermediate point they will initially show distance/time to the first intermediate, then when that point is reached they change to show the distance/time to the destination.  This would show progress along the route but taking up only two widget places instead of four.

Users who would like to see both the intermediate and destination information shown can configure the separate widgets as before.

### Describe the solution you'd like

2 additional widgets available in "Navigation points" showing information as described.

### Describe alternatives you've considered

Show the separate widgets, at a cost in screen space.

**Comment by vshcherb** [comment]
Yes, widget to final point could have an option to display to 1st intermediate point if it's available. We also could rename it to Navigation Point.

**Comment by dmpr0** [comment]
@vshcherb I think is better to use intermediate point widget for this purpose. We will have:
* Two widgets that will show Distance/Time to Destination point.
* Two widgets that will show Distance/Time to the Next Navigation Point.

**Comment by mikehgentry** [comment]
Two other good ones would be combined sunrise / sunset, and a combined current / average speed (shows average speed when you're not moving).

**Comment by ManDay** [comment]
> My suggestion is to have 2 more combined widgets available, showing the distance and time either to the next intermediate, or the final destination, **depending on what will arrive next.**

The formulation does not seem to make sense to me. The final destination will never arrive before the next intermediate. On the other hand, the *Distance to intermediate* widget currently simply vanishes if the next target is the final destination. I think the only useful distinction is

- **Distance (time) to final destination**
- **Distance (time) to next waypoint** (which can either be the final destination or an intermediate)

**Comment by DmitryAlexei** [comment]
Closed as duplicate of
- https://github.com/osmandapp/OsmAnd/issues/15479
